### DISCUSSION TITLE
Lock Down Pages By Group

### DISCUSSION THREAD

**Discussion by DanteCady** [requarks/wiki#7320] [Help / Questions] [answered]
Hi All,

Im working on my WikiJS site and wanted to know if it's possible to allow certain groups access to certain pages. I was looking in the admin panel and could not find anything around that. I have two groups of users who will have access to certain items and I want to ensure that each group can only access their respective content. 

Is this currently possible?

**Comment by NGPixel** [comment, marked as answer]
https://docs.requarks.io/groups
